app/frontend/src/lib/utils.js

```diff
@@ -61,3 +61,5 @@ export const storageAvailable = (type, logMessage = false) => {
           && (storage && storage.length !== 0);
   }
 };
+
+export const railsCsrfToken = () => document.getElementsByName('csrf-token')[0].content;
```

Comment:
does this need to export a function? `export const [railsCsrfToken] = blah;` also if there is no el with name csrf-token, it will fail trying to find `.content` of undefined

Reply:
I think it needs to be a function to actually have access to the document - or am I missing something?

Have rewritten it to fail less badly when the token isn't there (although it _should_ always be 🤞 )